### ISSUE TITLE
Add a module that assigns desired AMBER GB model to OpenMM System object

### ISSUE THREAD

**Issue by jchodera** [choderalab/yank#89]
We have determined that the simplest way to allow access to multiple AMBER GB models is to write a module that takes an OpenMM `System` object, strips out any GB forces, creates the appropriate (custom) GB Force object, and assigns parameters in a manner similar to [ParmEd](https://github.com/swails/ParmEd).

**Comment by Lnaden** [comment]
This is not really on the 1.0 lists we have discussed, moving to 2.0

**Comment by jchodera** [comment]
Yeah, we don't need this until we switch to a `simtk.openmm.app.ForceField` workflow.
